src/design/ButtonBar/index.tsx

```diff
@@ -15,6 +15,15 @@ export const ButtonBar: React.FC<HTMLAttributes<HTMLDivElement>> = ({
       "[&_button]:active:outline-none",
       "[&_button]:border [&_button]:focus:ring-0 [&_button]:focus:ring-offset-0",
       "[&_button]:active:ring-0 [&_button]:active:ring-offset-0",
+
```

Comment:
Can you clean up the comments here? There are some empty l ines and a 
`//in case of label`  comment that I don't get.